recipes/cairo/all/conanfile.py

```diff
@@ -75,18 +92,21 @@ def requirements(self):
 
     def build_requirements(self):
         if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
```

Comment:
Can you please add a `_settings_build` property method at the top (below `_build_subfolder`)
```python
    @property
    def _settings_build(self):
        return getattr(self, "settings_build", self.settings)
```
and do the following in `build_requirements`:
```python
        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
            self.build_requires("msys2/cci.latest")
```